apps/web/pages/index.tsx

```diff
@@ -8,7 +8,12 @@ export default function Web(): JSX.Element {
   const [loading, setLoading] = useState(true);
 
   useEffect(() => {
-    setLoading(false);
+    const access_token = localStorage.getItem("access_token");
+    if (access_token === null) {
+      router.replace("/login");
+    } else {
+      setLoading(false);
+    }
```

Comment:
This is handled by the _Routes_ component. This is probably due to the branch being based from a really old commit.